swagger2.0/oic.sec.ael.swagger.json

```diff
@@ -0,0 +1,263 @@
+{
+	"swagger": "2.0",
+	"info": {
+		"title": "Security Event List",
+		"version": "2019-10-03",
+		"license": {
+			"name": "OCF Data Model License",
+			"url": "https://github.com/openconnectivityfoundation/core/blob/e28a9e0a92e17042ba3e83661e4c0fbce8bdc4ba/LICENSE.md",
+			"x-copyright": "Copyright 2019 Open Connectivity Foundation, Inc. All rights reserved."
+		},
+		"termsOfService": "https://openconnectivityfoundation.github.io/core/DISCLAIMER.md"
+	},
+	"schemes": ["http"],
+	"consumes": ["application/json"],
+	"produces": ["application/json"],
+	"paths": {
+		"/AelResURI": {
+			"get": {
+				"description": "This Resource contains the Auditable Events that have been logged on the Device.",
+				"parameters": [
+					{ 
+						"$ref": "#/parameters/interface" 
+					},
+					{
+						"ref": "#/parameters/events-filtered-by-di"
+					},
+					{
+						"ref": "#/parameters/events-filtered-by-category"
+					}
+				],
+				"responses": {
+					"200": {
+						"description": "Example response payload. In this example, 'oic.d.light' Device has logged 2 Auditable Event Entries: Update attempt against '/room1/led1' Resource was denied, and Delete attempt against '/room1/led1' Resource was denied. Both Auditable Event Entries belong to 'AccessControl (0x01)' category and 'WARN' priority (2).",
+						"x-example": {
+							"rt": [ "oic.r.ael" ],
+							"events": [
+								{
+									"aeid": "AC-1",
+									"category": 1,
+									"priority": 2,
+									"timestamp": "2018-11-13T20:20:39+00:00",
+									"message":  "Access Denied",
+									"auxiliaryinfo": [ "0f33887b-f7d6-4fdb-9125-dd4b60d5aaae", "[2001::1]:1234", "/room1/led1", "U", "RFNOP", "No roles asserted" ]
+								},
+								{
+									"aeid": "AC-1",
+									"category": 1,
+									"priority": 2,
+									"timestamp": "2018-11-13T20:22:00+00:00",
+									"message":  "Access Denied",
+									"auxiliaryinfo": [ "0f33887b-f7d6-4fdb-9125-dd4b60d5aaae", "[2001::1]:1234", "/room1/led1", "D", "RFNOP", "No roles asserted" ]
+								}
+							],
+							"usedspace": 2,
+							"maxspace": 5
+						},
+						"schema": { "$ref": "#/definitions/Ael" }
+					}
+				}
+			}, 
+			"post": {
+				"description": "An UPDATE operation may set the 'categoryfilter' and/or 'priorityfilter' Properties.",
+				"parameters": [
+					{ 
+						"$ref": "#/parameters/interface" 
+					},
+					{
+						"in": "body",
+						"name": "body",
+						"required": false,
+						"schema": {
+							"$ref": "#/definitions/Ael"
+						},
+						"x-example": {
+							"categoryfilter": 3,
+							"priorityfilter": 1
+						}
+					}
+				],
+				"responses": {
+					"204": {
+						"description": "The new categoryfilter and priorityfilter were set."
+					}
+				}
+			}
+		}
+	},
+	"parameters": {
+		"interface": {
+			"in": "query",
+			"name": "if",
+			"type": "string",
+			"enum": [
+				"oic.if.baseline",
+				"oic.if.r"
```

Comment:
This should be "oic.if.rw" since the Resource supports UPDATE.

Reply:
Fixed in https://github.com/openconnectivityfoundation/security/pull/120